packages/lesswrong/components/sequences/SequencesHoverOver.tsx

```diff
@@ -74,13 +74,16 @@ export const SequencesHoverOver = ({classes, sequence, showAuthor=true}: {
     </ContentStyles>
     {/* show a loading spinner if either sequences hasn't loaded or chapters haven't loaded */}
     {(!sequence || (!chapters && chaptersLoading)) && <Loading/>}
-    {sequence && posts.map(post => 
-      <SequencesSmallPostLink 
-        key={sequence._id + post._id} 
-        post={post}
-        sequenceId={sequence._id}
-      />
-    )}
+    {sequence && chapters?.flatMap(chapter => {
```

Comment:
`chapters?.map` should be sufficient, I think?  We aren't returning an array that we want to flatten from inside it